custom_components/adjustable_bed/beds/okin_uuid.py

```
@@ -0,0 +1,673 @@
+
+    async def write_command(
+        self,
+        command: bytes,
+        repeat_count: int = 1,
+        repeat_delay_ms: int = 100,
+        cancel_event: asyncio.Event | None = None,
+    ) -> None:
+        """Write a command to the bed."""
+        if self.client is None or not self.client.is_connected:
+            _LOGGER.error("Cannot write command: BLE client not connected")
+            raise ConnectionError("Not connected to bed")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)